src/components/BulkEditStartModal/BulkEditStartModal.js

```diff
@@ -40,15 +50,47 @@ const BulkEditStartModal = ({
     setDropZoneActive(false);
   };
 
+  const uploadFileFlow = async (fileToUpload) => {
+    setFileName(fileToUpload.name);
+    setDropZoneActive(false);
+
+    try {
+      const { id } = await requestJobId({ recordIdentifier: 'BARCODE', editType: BULK_EDIT_UPDATE });
```

Comment:
may we move recordIdentifiers in constants?

Reply:
I have already moved it. Thanks for your suggestion.